contracts/Auction.sol

```diff
@@ -32,6 +32,7 @@ contract Auction is IAuction {
         Auctioneer auctioneer;
         // the auction price, denominated in tokenAccepted
         uint256 amountOutstanding;
+        uint256 amountInitial;
```

Comment:
Maybe `amountDesired`? The desired amount does not change, only the outstanding amount does.

Reply:
Also done in https://github.com/keep-network/coverage-pools/pull/43/commits/ba3de981619c6aea018188c28b64fc2e99f4ce49